test/test_utils/ec2.py

```diff
@@ -88,7 +88,7 @@ def launch_instance(
     return response["Instances"][0]
 
 
-def ec2_client(region):
+def get_ec2_client(region):
     return boto3.client("ec2", region_name=region, config=Config(retries={'max_attempts': 10}))
```

Comment:
Is this function not used anywhere?

Reply:
for DLC_tests we are using the one defined in the contest.py. I haven't used this for sm_local